src/pages/VoucherRedemption/VoucherRedemption.tsx

```diff
@@ -28,6 +29,29 @@ const VoucherRedemption = (props: Props) => {
   const [loading, setLoading] = useState(false);
 
   const fetchData = async () => {
+    let getLocationInfo = (merchantData): LocationInfo => {
+      const emptyLocationInfo = {
+        line1: '',
+        line2: '',
+      };
+      if (!merchantData || !merchantData.data) {
```

Comment:
You might be able to use [Optional Chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) here to make this a little bit cleaner if you want. Could combine all of these checks below into one or a few statements.

Reply:
ooh nice! Did not know about this